dev/build_release.py

```diff
@@ -371,11 +371,24 @@ def __gcb_build(cls, name, gradle_root, gcb_service_account, gcb_project):
     if os.path.isdir(gradle_cache):
       # Tell rmtree to delete the directory even if it's non-empty.
       shutil.rmtree(gradle_cache)
-    cmds = [
-      ('gcloud container builds submit --log-http'
-       ' --account={account} --project={project} --config="../{name}-gcb.yml" .'
-       .format(name=name, account=gcb_service_account, project=gcb_project))
-    ]
+    if self.__options.externalize_gcs_upload:
+      tar_filename = name + '-{:%Y%m%d%H%M%S}'.format(datetime.datetime.utcnow())
+      gcs_staging_bucket = self.__options.gcs_staging_bucket
+      cmds = [
+        ('tar -cf {tar_filename}.tar .'
+          .format(tar_filename=tar_filename)),
+        ('gsutil cp {tar_filename}.tar {gcs_staging_bucket}'
+          .format(tar_filename=tar_filename, gcs_staging_bucket=gcs_staging_bucket)),
```

Comment:
You can have an account conflict here between gsutil and gcloud.

Reply:
gsutil doesnt let you set the account, so you'll need to gcloud set account before running this feature (which will have the side effect of changing the account for gcloud after the command is run).

Reply:
I added the `gcloud config set account ...` call. I don't see a way out of explicitly doing this, and all subsequent gcloud calls are specifying the --account parameter. If you have alternative methods, please suggest.

Reply:
as discussed offline, call is put in the caller of this script rather than hidden in this script to avoid subtle permanent side effects leading to confusion and possible bugs. the change is still made, but by a different role (that already manages accounts) than this script so should be more clear that it is happening.